client/lib/domains/registration/availability-messages.js

```diff
@@ -647,6 +647,13 @@ function getAvailabilityNotice(
 			severity = 'info';
 			break;
 
+		case 'hundred_year_domain_tld_restriction':
+			message = translate(
+				'Only .com, .net and .org domains are available for registration for 100 years.'
```

Comment:
Not sure if this was provided, but I might suggest 

“Only .com, .net, and .org domains can be registered for a 100-year period.”
or
“Only .com, .net, and .org domains can be registered 100 years.”

Reply:
Good suggestions! I'll update this